### DISCUSSION TITLE
CRC in SX1278 Module

### DISCUSSION THREAD

**Discussion by Jaimi5** [jgromes/RadioLib#486] [Q&A] [answered]
I have some questions about the use of the CRC using LoRa and SX1278 Module.
When activating the radio.setCRC(true), you will send 15 bits more of data through LoRa? Then the maximum payload that you could send is 255 bytes - 15 bites = 253 bytes + 1 bit?
And using the CRC-16-CCITT or CRC-16-IBM need to use a even number of bits will result of a 253 bytes of payload?

 I hope the question is not absurd and thanks for your time 😊!

**Comment by jgromes** [comment, marked as answer]
> When activating the radio.setCRC(true), you will send 15 bits more of data through LoRa?

You will be sending two extra bytes (16 bits) at the end of the packet. However, this has no impact on the maximum payload length, that is always limited to 255 bytes. I don't really understand where the 15 bits came from.

> And using the CRC-16-CCITT or CRC-16-IBM need to use a even number of bits will result of a 253 bytes of payload?

Not sure I understand the question. You can not send an odd number of bits, because the minimum packet length unit is a byte (8 bits). You can't send half a byte.

**Reply by Jaimi5** [reply]
I am doing a LoRa Mesh network and it was just to understand how many bytes for each packet it sends!

Thanks for the quick response, that's all I needed!
